spec/e2e_spec.coffee

```diff
@@ -166,6 +166,20 @@ describe 'JUnit Report builder', ->
       '    </testcase>\n' +
       '  </testsuite>')
 
+  it 'should print the reported attachment to system-err', ->
+    builder.testSuite().testCase()
+      .standardError("This was written to stderr!")
+      .errorAttachment("absolute/path/to/attachment")
+
+    expect(builder.build()).toBe reportWith(
+      '  <testsuite tests="1" failures="0" errors="0" skipped="0">\n' +
+      '    <testcase>\n' +
+      '      <system-err>\n' +
+      '        <![CDATA[This was written to stderr!]]>\n' +
+      '        [[ATTACHMENT|absolute/path/to/attachment]]\n' +
```

Comment:
I'm not able to verify that this actually works as expected on Jenkins. Have you tried it?

Reply:
The truth is that I haven't tried it. I have implemented the spec by the book. I could find a way to test it on Jenkins but it'd take me some time!

Reply:
That would be great! I wouldn't feel that good if I released something that isn't working, and I don't really have the time to set up an environment to try this out myself.